tests/security/test_secret.py

```diff
@@ -78,11 +81,14 @@ def test_token(db, config, encrypt):
     if encrypt:
         assert secret._secret != token, "token not encrypted"
     else:
-        assert secret._secret == token, "token encrypted when encryption disabled"
+        assert secret._secret == token.encode(
```

Comment:
on Python 3 `bytes` cannot be compared with `str` so you need explicit `encode()`